images/kcp-registrar/register.sh

```diff
@@ -214,12 +214,30 @@ register() {
                 --syncer-image "ghcr.io/kcp-dev/kcp/syncer:$KCP_SYNC_TAG" \
                 --resources deployments.apps,services,ingresses.networking.k8s.io,pipelines.tekton.dev,pipelineruns.tekton.dev,tasks.tekton.dev,runs.tekton.dev,networkpolicies.networking.k8s.io \
                 --output-file "$syncer_manifest"
+            add_ca_cert_to_syncer_manifest "${kcp_kcfg}" "$syncer_manifest"
```

Comment:
That seems strange to me, we should not have to mess with the manifest generated by kcp to get a secure connection.
If adding a ca_cert is helpful/requried, it seems that this should be implemented on kcp side.

Reply:
I think that's true only if the underlying cluster itself uses a valid identity cert which is not the case in most development environments.

Reply:
Any chance the certificate we're trying to extract might not exists? I don't see any error handling around that, which is fine with me if we know that the certificate should always be there on a properly configured cluster.

Reply:
I think we can assume that it'll always be there because we're querying the OpenShift API which is always served via HTTPS. If a cert does not exist, the cluster is likely non-functional.